pkg/chain/ethereum/beacon.go

```diff
@@ -83,13 +100,28 @@ func newBeaconChain(
 		)
 	}
 
+	chainConfig, err := fetchChainConfig(randomBeacon)
```

Comment:
Eventually, we will probably need to fetch some values from the `RandomBeacon` contract. For example, relay entry timeouts.

Reply:
I am not sure about it. The idea with `GetConfig` was that the values do not change. Timeouts can change for the running client, so I think they will have to be fetched at the moment of submission, with a separate function to get only one information from the chain. For example, "fetch me the current value of relay entry timeout" instead of fetching authorization decrease timeout, DKG timeout, group size, and all other parameters.

Reply:
That makes sense! See 632ef377